dts/bindings/stepper/adi/adi,tmc2209.yaml

```diff
@@ -23,13 +23,9 @@ include:
       - micro-step-res
       - step-gpios
       - dir-gpios
+      - en-gpios
 
 properties:
-  enable-gpios:
-    type: phandle-array
-    description: |
-      GPIO pins used to control the enable signal of the motor driver.
```

Comment:
since this property is expected to be user (board designer) facing, maybe the migration guide needs to be updated?